wbooks/src/app/screens/SearchScreen/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native';
+import { useSelector } from 'react-redux';
+import BookList from '@components/BookList';
+import SearchEmpty from '@components/SearchEmpty';
+import useFilteredBooks from '@hooks/useFilteredBooks';
+import { BookState } from '@interfaces/bookState';
+import { RootState } from '@interfaces/rootState';
+
+import styles from './styles';
+
+function SearchScreen() {
+  const filteredBooks = useFilteredBooks();
+
+  const { searchedBook } = useSelector<RootState, BookState>(state => state.books);
```

Comment:
Use the Hook to get this value => `const [searchedBook, filteredBooks] = useFilteredBooks();`

Reply:
I renamed the hook as `useBooks` and updated it so it returns `searchedBook`, `filteredBooks` and `books` (as in all books) to make it even more reusable, I kept the return type as an object, as it makes it easier to ignore values that are not needed.